src/boot/bootctl-install.c

```diff
@@ -423,7 +428,7 @@ static int copy_one_file(const char *esp_path, const char *name, bool force) {
                 /* If we were installed under any other name in /EFI/BOOT, make sure we update those binaries
                  * as well. */
                 if (!force)
-                        RET_GATHER(ret, update_efi_boot_binaries(esp_path, source_path));
+                        RET_GATHER(ret, update_efi_boot_binaries(esp_path, source_path, default_dest_path));
```

Comment:
hmm, i find it weird to pass the full path here, and then throwing it away in the calleee. I think it would make more sense to pass in only he file name.